app/scripts/lib/ppom/ppom-util.ts

```diff
@@ -210,7 +221,7 @@ async function validateWithController(
 async function validateWithAPI(
   ppomController: PPOMController,
   chainId: string,
-  request: JsonRpcRequest,
+  request: SecurityAlertsAPIRequest | JsonRpcRequest,
```

Comment:
Same as above, the output from `normalizePPOMRequest` is passed in as the `request` argument in [`validateRequestWithPPOM`](https://github.com/MetaMask/metamask-extension/pull/26143/files#diff-56c892dd2b3324d3e12096b6e2280dafd007572a2c6797f8f6edc9a6f782b293R55).